### ISSUE TITLE
sbt doesn't see new SNAPSHOT dependency versions in local ivy2 repo

### ISSUE THREAD

**Issue by denisgarci** [sbt/sbt#4276]
- [X ] I've read the [CONTRIBUTING](https://github.com/sbt/sbt/blob/1.x/CONTRIBUTING.md) guidelines

## problem
I'm using sbt with a local ivy2 repo (e.g. `~/.ivy2/local`) for a Scala project that depends on a SNAPSHOT version of Scala library that is being developed at the same time.
I published the library using `publishLocal`. But when I run a `clean update` on the project only the SNAPSHOT version hosted on Nexus is chosen with the following warning:
```
[warn] Choosing *** Nexus for com.org#library_2.11;version-SNAPSHOT
```

## expectation
The local SNAPSHOT should be selected because it's more recent.

## notes
Similar too https://github.com/sbt/sbt/issues/321 but with local ivy2 repo instead.

sbt version: 1.1.6

**Comment by eed3si9n** [comment]
Do you see publication field in your ivy.xml, and lastUpdated in maven-metadata.xml?
If yes, what are the values?

**Comment by denisgarci** [comment]
Thanks for the quick reply. Yes I can see the `<publications>` in the `ivy.xml`. I couldn't find a file called `maven-metadata.xml`. Where should I look for it?

**Comment by michaelahlers** [comment]
I have this exact same problem with SBT 1.1.6 and 1.2.1.

**Comment by KarelCemus** [comment]
Any progress on a solution to this issue?

**Comment by eed3si9n** [comment]
I think this will be resolved using sbt 1.3.0-RC3, which uses Coursier internally, but I'd caution against any use of non-local SNAPSHOTs.
